eng/common/scripts/Create-APIReview.ps1

```diff
@@ -41,6 +51,13 @@ function Submit-APIReview($packagename, $filePath, $uri, $apiKey, $apiLabel, $re
     $versionContent.Headers.ContentDisposition = $versionParam
     $multipartContent.Add($versionContent)
     Write-Host "Request param, packageVersion: $packageVersion"
+    
+    $releaseTagParam = [System.Net.Http.Headers.ContentDispositionHeaderValue]::new("form-data")
+    $releaseTagParam.Name = "setReleaseTag"
```

Comment:
I assume this is the name of the API in APIView? Ideally we update it as well but I understand that means an update to the service.

Reply:
Correct.